www/src/pages/blog/tags.js

```diff
@@ -39,8 +39,11 @@ const POPULAR_TAGS = [
 const PopularTagGrid = styled.div`
   display: grid;
   grid-auto-rows: 1fr;
-  grid-template-columns: repeat(3, 1fr);
+  grid-template-columns: repeat(2, 1fr);
```

Comment:
It might be more flexible to use the technique described here: https://css-tricks.com/look-ma-no-media-queries-responsive-layouts-using-css-grid/

That way we're not relying on fixed media queries and specific column widths.

Reply:
Yeah Sure. I will convert it without using media queries

Reply:
@tesseralis I have made the changes. Now no media query is being used. Tags automatically adjust themselves according to the screen width without breaking the UI.